README.md

```diff
@@ -57,31 +58,30 @@ app.listen(3000);
 
 You're defining an interceptor that will be executed whenever the response contains html as Content-Type. If this is true, it will append a child at the end of the body.
 
-Find other examples at [/examples folder](https://github.com/axiomzen/express-interceptor/tree/master/examples).
+See [more examples](https://github.com/axiomzen/express-interceptor/tree/master/examples).
 
 ## API
 
-* * `isInterceptable()`: is a predicate function where you define a condition whether or not to intercept a response. Returning `true` buffers the request, and proceeds calling `intercept()` as well as `afterSend()`. Typically, you want to check for this condition in the `res` object in the definition of the middleware.
+* `isInterceptable()` (required): is a predicate function where you define a condition whether or not to intercept a response. Returning `true` buffers the request, and proceeds calling `intercept()` as well as `afterSend()`. Typically, you want to check for this condition in the `res` object in the definition of the middleware.
```

Comment:
The `*` were being interpreted as a nested list. Adding (required) is more explicit.